lib/src/common/nas_pcap.cc

```diff
@@ -22,10 +22,22 @@
 #include "srsran/common/nas_pcap.h"
 #include "srsran/common/pcap.h"
 #include "srsran/srsran.h"
+#include "srsran/support/emergency_handlers.h"
 #include <stdint.h>
 
 namespace srsran {
 
+/// Try to flush the contents of the pcap class before the application is killed.
+static void emergency_cleanup_handler(void* data)
+{
+  reinterpret_cast<nas_pcap*>(data)->close();
+}
+
+nas_pcap::nas_pcap() : enable_write(false), ue_id(0), pcap_file(NULL)
```

Comment:
@andrepuschmann I moved variable initialization here, that should be equivalent?

Reply:
it's equivalent. but the current code policy is to init them in the header as well to default values. Would be great if you could follow that. thanks